controllers/array_action/array_mediator_svc.py

```diff
@@ -1463,17 +1665,12 @@ def _add_snapshot(self, snapshot_name, source_id, pool):
             raise array_errors.ObjectNotFoundError(snapshot_id)
         return cli_snapshot
 
-    def _mkvolumegroup(self, name, pool, io_group, source_id):
-        cli_kwargs = {
-            'type': 'clone',
-            'fromsnapshotid': source_id,
-            'pool': pool,
-            'name': name
-        }
-        if io_group:
-            cli_kwargs['iogroup'] = io_group
+    def _mkvolumegroup(self, **cli_kwargs):
+        name = cli_kwargs['name']
+        pool = cli_kwargs['pool'] if 'pool' in cli_kwargs else None
         try:
-            return self.client.svctask.mkvolumegroup(**cli_kwargs)
+            svc_response = self.client.svctask.mkvolumegroup(**cli_kwargs)
```

Comment:
if name should pass anyway so maybe we can rewrite like:
```
def _mkvolumegroup(self, name, **cli_kwargs):
        pool = cli_kwargs['pool'] if 'pool' in cli_kwargs else None
        try:
            svc_response = self.client.svctask.mkvolumegroup(name=name, **cli_kwargs)
```
like with [_chvdisk](https://github.com/IBM/ibm-block-csi-driver/blob/b79cf33965994ea5d5e00696cab518d8fd4c12eb/controllers/array_action/array_mediator_svc.py#L1719)

Reply:
Done